package.json

```diff
@@ -44,76 +44,76 @@
-    "ember-cli-htmlbars": "^5.7.1",
-    "ember-destroyable-polyfill": "^2.0.3"
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-htmlbars": "^6.1.1"
   },
   "devDependencies": {
     "@babel/cli": "^7.18.9",
+    "@babel/core": "^7.20.5",
     "@babel/preset-typescript": "^7.18.6",
     "@ember/optional-features": "^2.0.0",
-    "@glimmer/component": "^1.0.4",
+    "@glimmer/component": "^1.1.2",
     "@glimmer/interfaces": "^0.84.1",
     "@glimmer/reference": "^0.84.1",
     "@tsconfig/ember": "^1.0.1",
     "@types/ember": "~4.0.0",
+    "@types/ember-testing-helpers": "^0.0.4",
     "@types/ember__application": "~4.0.0",
     "@types/ember__component": "~4.0.8",
     "@types/ember__debug": "~4.0.1",
     "@types/ember__destroyable": "~4.0.0",
     "@types/ember__engine": "~4.0.0",
     "@types/ember__object": "~4.0.2",
-    "@types/ember__routing": "~4.0.7",
     "@types/ember__polyfills": "~4.0.0",
+    "@types/ember__routing": "~4.0.7",
     "@types/ember__runloop": "~4.0.1",
     "@types/ember__service": "~4.0.0",
     "@types/ember__test": "~4.0.0",
-    "@types/ember-testing-helpers": "^0.0.4",
     "@types/rsvp": "^4.0.4",
     "@typescript-eslint/eslint-plugin": "^4.29.0",
     "@typescript-eslint/parser": "^4.29.0",
     "broccoli-babel-preset-typescript": "^1.0.1",
     "broccoli-merge-trees": "^4.2.0",
     "documentation": "^13.2.5",
-    "ember-auto-import": "^2.2.0",
-    "ember-cli": "~3.28.3",
-    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-auto-import": "^2.5.0",
+    "ember-cli": "~4.8.0",
+    "ember-cli-dependency-checker": "^3.3.1",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-test-loader": "^3.0.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-fetch": "^8.1.0",
     "ember-in-element-polyfill": "^1.0.1",
     "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator-for-testing": "^1.0.0",
-    "ember-resolver": "^8.0.2",
-    "ember-source": "~3.28.4",
+    "ember-resolver": "^8.0.3",
+    "ember-source": "~4.9.0",
     "ember-source-channel-url": "^3.0.0",
-    "ember-try": "^1.4.0",
+    "ember-try": "^2.0.0",
     "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
+    "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-disable-features": "^0.1.3",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-prettier": "^4.2.1",
     "expect-type": "^0.13.0",
     "fs-extra": "^10.0.0",
     "latest-version": "^5.0.0",
     "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
     "pretender": "^3.4.7",
-    "prettier": "^2.4.1",
-    "qunit": "^2.19.1",
+    "prettier": "^2.8.0",
+    "qunit": "^2.19.3",
     "release-it": "~14.11.6",
     "release-it-lerna-changelog": "^3.1.0",
     "rimraf": "^3.0.2",
     "typescript": "^4.7.4",
     "webpack": "^5.74.0"
   },
   "engines": {
-    "node": "10.* || 12.* || 14.* || 15.* || >= 16.*"
+    "node": "14.* || 16.* || >= 18"
```

Comment:
I think it's coming from ember-source as well.

`ember-source@4.9.0: The engine "node" is incompatible with this module. Expected version ">= 14.*". Got "12.22.4"`

Reply:
For posterity: I made the change, which landed in Ember 4.9, to drop support for older versions of Node in Ember itself… and had forgotten it when I made this comment.

What I think I’m going to do is cut a release of the current major version of `@ember/test-helpers` which dos *not* include this particular update, but *does* ship types. That'll do what we *intended* to do months ago (though apparently no one has been depending on that, since we haven't gotten bug reports!). At that point I'll also update DefinitelyTyped to match.

Then *this* PR, and shipping native Ember types, can go into a new major version.